app/models/location.rb

```diff
@@ -139,7 +139,8 @@ def status
 
   def status_at(date)
     return :incomplete unless confirmed
-    return :deactivated if open_deactivation&.deactivation_date.present? && date >= open_deactivation.deactivation_date
+    return :deactivated if scheme.owning_organisation.status_at(date) == :deactivated ||
+      open_deactivation&.deactivation_date.present? && date >= open_deactivation.deactivation_date
```

Comment:
There's some awkward code here, cause we also have a `deactivated` scope in this file, which is independent of this and probably needs updating